packages/components/src/hooks/use-constructor.ts

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-export const useConstructor = (callBack = () => {}) => {
+export const useConstructor = (callBack = () => undefined) => {
```

Comment:
`callBack = () => {}` this threw an error that empty arrow functions cannot be used. Hence changed the `{}` to undefined

Reply:
in this case, we have to declare the callback type, otherwise TS will think that we can accept here only the callbacks which return `undefined`:
<img width="662" alt="Screenshot 2022-10-07 at 17 41 30" src="https://user-images.githubusercontent.com/103177211/194580405-30e5c86b-9951-4ea8-8e54-b12a996ef2ab.png">
```suggestion
export const useConstructor = (callBack: () => void = () => undefined) => {
```
🙂